docs/pages/index.tsx

```diff
@@ -48,7 +48,7 @@ export default function IndexPage() {
       <MWrapper>
         <IntroWrapper>
           <IntroHeading>
-            The <Highlight grad="grad1">superpowered</Highlight> CMS for developers
+            The <Highlight look="grad1">superpowered</Highlight> CMS for developers
```

Comment:
I fixed that already in my branch... why is it not picking that up?

Reply:
See here: https://github.com/keystonejs/keystone/blob/dom/site/docs/pages/index.tsx#L51